test/services/news/index.test.js

```diff
@@ -154,6 +154,40 @@ describe('news service', () => {
 				}
 			});
 
+			it('should respond with an error if a student is trying to access specific unpublished news', async () => {
+				const schoolId = (await createTestSchool())._id;
+				const now = Date.now();
+				const weekInMilSeconds = 604800000; // 1000 * 60 * 60 * 24 * 7
+				const user = await createTestUser({
+					schoolId,
+					roles: 'student',
+				});
+
+				const user2 = await createTestUser({
+					schoolId,
+					roles: 'administrator',
+				});
+
+				const schoolNews = await News.create({
+					creatorId: user2._id,
+					schoolId,
+					title: 'hacker news',
+					content:
+						"I can't access news articles that are not public yet",
+					displayAt:
+						now + weekInMilSeconds,
+				});
+				const params = await generateRequestParamsFromUser(user);
+
+				try {
+					await newsService.get(schoolNews._id, params);
```

Comment:
the admin user should be able to fetch it, test it :)

Reply:
added an finally clause to check if admin can see the unpublished news
´
...
finally {
					const result = await newsService.get(schoolNews._id, adminParams);
					expect(result.title).to.be.equal('hacker news')
				}
...
´